code/modules/mining/mine_turfs.dm

```diff
@@ -619,10 +619,12 @@
 
 /turf/open/floor/plating/asteroid/basalt/lava_land_surface
 	initial_gas_mix = "o2=14;n2=23;TEMP=300"
+	planetary_atmos = TRUE
 	baseturf = /turf/open/floor/plating/lava/smooth/lava_land_surface
 
 /turf/open/chasm/straight_down/lava_land_surface
 	initial_gas_mix = "o2=14;n2=23;TEMP=300"
+	planetary_atmos = TRUE
```

Comment:
Given that /turf/open/chasm/straight_down/lava_land_surface/normal_air is only used on the Snappop shuttle at the moment, it'd be weird for it to have planetary_atmos on a ship. How does planetary atmos interact with air leaking to space?